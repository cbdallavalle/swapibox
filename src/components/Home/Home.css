.Home {
  width: 270px;
  margin: auto;
  background-color: #ffffff1f;
}

p {
  margin: 0;
}