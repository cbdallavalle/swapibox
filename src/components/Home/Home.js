import React from 'react';
import './Home.css';

const Home = () => {
  return (
    <article className="Home">
      <p> 
        It is a period of civil war. 
        Rebel spaceships, striking from a hidden base, have won their first victory 
        against the great Galactic Empire. During the battle, Rebel scum spies managed 
        to steal secret plans to the Empire’s ultimate weapon, the DEATH STAR, 
        an armored space station with enough power to destroy an entire planet.

        Pursued by the Empire’s finest agents, 
        Princess Leia races home aboard her starship, 
        custodian of the stolen plans that can enslave that rebel scum 
        and restore order to the galaxy… 
      </p>
    </article>
  )
}

export default Home;