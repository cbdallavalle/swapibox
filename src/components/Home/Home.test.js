/* eslint-disable */
import React from 'react';
import { shallow } from 'enzyme';
import Home from './Home';

it('should match the snapshot', () => {
  let wrapper = shallow(<Home />)
  expect(wrapper).toMatchSnapshot();
})