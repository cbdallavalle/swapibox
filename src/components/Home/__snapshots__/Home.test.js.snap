// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Home
    filmCrawl={
        Object {
            "crawl": "It is a period of civil war.
        Rebel scum spaceships, striking
        from a hidden base, have won
        their first and only victory against
        the great Galactic Empire.
        
        During the battle, Rebel
        scum spies managed to steal secret
        plans to the Empire's
        ultimate weapon, the DEATH
        STAR, an armored space
        station with enough power
        to destroy an entire planet.
        
        Pursued by the Empire's
        finest agents, Princess
        Leia flees home aboard her
        starship, custodian of the
        stolen plans that can enslave her
        people and restore
order to the galaxy....",
            "date": "1977-05-25",
            "title": "No Hope",
          }
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="crawl"
>
        <h1>
                No Hope
        </h1>
        <h2>
                1977-05-25
        </h2>
        <p>
                It is a period of civil war.
                Rebel scum spaceships, striking
                from a hidden base, have won
                their first and only victory against
                the great Galactic Empire.
                
                During the battle, Rebel
                scum spies managed to steal secret
                plans to the Empire's
                ultimate weapon, the DEATH
                STAR, an armored space
                station with enough power
                to destroy an entire planet.
                
                Pursued by the Empire's
                finest agents, Princess
                Leia flees home aboard her
                starship, custodian of the
                stolen plans that can enslave her
                people and restore
order to the galaxy....
        </p>
</div>,
      "className": "Home",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            No Hope
</h1>,
          <h2>
            1977-05-25
</h2>,
          <p>
            It is a period of civil war.
            Rebel scum spaceships, striking
            from a hidden base, have won
            their first and only victory against
            the great Galactic Empire.
            
            During the battle, Rebel
            scum spies managed to steal secret
            plans to the Empire's
            ultimate weapon, the DEATH
            STAR, an armored space
            station with enough power
            to destroy an entire planet.
            
            Pursued by the Empire's
            finest agents, Princess
            Leia flees home aboard her
            starship, custodian of the
            stolen plans that can enslave her
            people and restore
order to the galaxy....
</p>,
        ],
        "className": "crawl",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "No Hope",
          },
          "ref": null,
          "rendered": "No Hope",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "1977-05-25",
          },
          "ref": null,
          "rendered": "1977-05-25",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "It is a period of civil war.
Rebel scum spaceships, striking
from a hidden base, have won
their first and only victory against
the great Galactic Empire.

During the battle, Rebel
scum spies managed to steal secret
plans to the Empire's
ultimate weapon, the DEATH
STAR, an armored space
station with enough power
to destroy an entire planet.

Pursued by the Empire's
finest agents, Princess
Leia flees home aboard her
starship, custodian of the
stolen plans that can enslave her
people and restore
order to the galaxy....",
          },
          "ref": null,
          "rendered": "It is a period of civil war.
Rebel scum spaceships, striking
from a hidden base, have won
their first and only victory against
the great Galactic Empire.

During the battle, Rebel
scum spies managed to steal secret
plans to the Empire's
ultimate weapon, the DEATH
STAR, an armored space
station with enough power
to destroy an entire planet.

Pursued by the Empire's
finest agents, Princess
Leia flees home aboard her
starship, custodian of the
stolen plans that can enslave her
people and restore
order to the galaxy....",
          "type": "p",
        },
      ],
      "type": "div",
    },
    "type": "article",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="crawl"
>
          <h1>
                    No Hope
          </h1>
          <h2>
                    1977-05-25
          </h2>
          <p>
                    It is a period of civil war.
                    Rebel scum spaceships, striking
                    from a hidden base, have won
                    their first and only victory against
                    the great Galactic Empire.
                    
                    During the battle, Rebel
                    scum spies managed to steal secret
                    plans to the Empire's
                    ultimate weapon, the DEATH
                    STAR, an armored space
                    station with enough power
                    to destroy an entire planet.
                    
                    Pursued by the Empire's
                    finest agents, Princess
                    Leia flees home aboard her
                    starship, custodian of the
                    stolen plans that can enslave her
                    people and restore
order to the galaxy....
          </p>
</div>,
        "className": "Home",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h1>
              No Hope
</h1>,
            <h2>
              1977-05-25
</h2>,
            <p>
              It is a period of civil war.
              Rebel scum spaceships, striking
              from a hidden base, have won
              their first and only victory against
              the great Galactic Empire.
              
              During the battle, Rebel
              scum spies managed to steal secret
              plans to the Empire's
              ultimate weapon, the DEATH
              STAR, an armored space
              station with enough power
              to destroy an entire planet.
              
              Pursued by the Empire's
              finest agents, Princess
              Leia flees home aboard her
              starship, custodian of the
              stolen plans that can enslave her
              people and restore
order to the galaxy....
</p>,
          ],
          "className": "crawl",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "No Hope",
            },
            "ref": null,
            "rendered": "No Hope",
            "type": "h1",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "1977-05-25",
            },
            "ref": null,
            "rendered": "1977-05-25",
            "type": "h2",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "It is a period of civil war.
Rebel scum spaceships, striking
from a hidden base, have won
their first and only victory against
the great Galactic Empire.

During the battle, Rebel
scum spies managed to steal secret
plans to the Empire's
ultimate weapon, the DEATH
STAR, an armored space
station with enough power
to destroy an entire planet.

Pursued by the Empire's
finest agents, Princess
Leia flees home aboard her
starship, custodian of the
stolen plans that can enslave her
people and restore
order to the galaxy....",
            },
            "ref": null,
            "rendered": "It is a period of civil war.
Rebel scum spaceships, striking
from a hidden base, have won
their first and only victory against
the great Galactic Empire.

During the battle, Rebel
scum spies managed to steal secret
plans to the Empire's
ultimate weapon, the DEATH
STAR, an armored space
station with enough power
to destroy an entire planet.

Pursued by the Empire's
finest agents, Princess
Leia flees home aboard her
starship, custodian of the
stolen plans that can enslave her
people and restore
order to the galaxy....",
            "type": "p",
          },
        ],
        "type": "div",
      },
      "type": "article",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
